app/src/main/res/layout/fragment_user_goals_init.xml

```diff
@@ -0,0 +1,122 @@
+            android:layout_gravity="left"
+            android:layout_marginLeft="5dp"
+            android:textSize="15sp"
+            android:text="@string/how_many_kilometer_do_you_want_to_run_through_each_day"
+            android:textColor="@color/design_default_color_surface"/>
+        <EditText
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_horizontal"
+            android:textSize="20sp"
+            android:id="@+id/input_distanceGoal_text_UserProfileCreation"
+            android:hint="@string/enter_your_distance_goal"
+            android:background="@color/design_default_color_surface" />
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_horizontal"
+            android:textSize="10dp"
+            android:id="@+id/distanceGoalError_text_userProfileCreation" />
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="left"
+            android:layout_marginLeft="5dp"
+            android:textSize="15sp"
+            android:text="@string/how_many_time_do_you_want_to_practice_exercise_each_day_in_minutes"
+            android:textColor="@color/design_default_color_surface"/>
+        <EditText
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_horizontal"
+            android:textSize="20sp"
+            android:id="@+id/input_timeGoal_text_UserProfileCreation"
+            android:hint="@string/enter_your_time"
+            android:background="@color/design_default_color_surface" />
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_horizontal"
+            android:textSize="10dp"
+            android:id="@+id/timeGoalError_text_userProfileCreation" />
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="left"
+            android:layout_marginLeft="5dp"
+            android:textSize="15sp"
+            android:text="@string/how_many_path_do_you_want_to_draw_each_day"
+            android:textColor="@color/design_default_color_surface"/>
+        <EditText
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_horizontal"
+            android:layout_marginBottom="10dp"
+            android:textSize="20sp"
+            android:id="@+id/input_nbOfPathsGoal_text_UserProfileCreation"
+            android:hint="@string/enter_your_number_of_path"
+            android:background="@color/design_default_color_surface"
+            />
```

Comment:
`android:inputType="number"` add this so that only number can be typed

You could also add these so that it stays on 1 line
`android:maxLines="1"`
`android:lines="1"`
`android:singleLine="true"`